### DISCUSSION TITLE
[Detector Support]: Openvino Width Height Model Options 300x300

### DISCUSSION THREAD

**Discussion by moogoos** [blakeblackshear/frigate#14825] [Detector Support] [answered]
### Describe the problem you are having

I am curious what the following options are and if there's any benefit to changing them. Are they baked into the config in the container? Is there any value for higher resolution here? My coral was 5fps 1280x720, I tried looking at the docs but couldn't find anything. Should these match by sub res? Does it matter? I can afford more GPU respources if needed.

### Version

15-beta

### Frigate config file

```yaml
model:
  width: 300
  height: 300
```
```


### docker-compose file or Docker CLI command

```yaml
NA
```

### Relevant Frigate log output

```shell
NA
```

### Install method

Docker Compose

### Object Detector

OpenVino

### Screenshots of the Frigate UI's System metrics pages

NA

**Comment by NickM-27** [comment, marked as answer]
These MUST match the model, which only supports one size input. You can not change them.
